nixos/modules/services/databases/postgresql.nix

```
@@ -428,7 +454,43 @@ in
           PostgreSQL superuser account to use for various operations. Internal since changing
           this value would lead to breakage while setting up databases.
         '';
-        };
+      };
+
+      upgradeFrom = mkOption {
+        type = types.nullOr (types.submodule ({ config, ... }: {
+          options = {
+            package = mkOption {
+              type = types.package;
+              description = lib.mdDoc "Package to upgrade from";
+            };
+            dataDir = mkOption {
+              type = types.path;
+              default = "/var/lib/postgresql/${config.package.psqlSchema}";
+              defaultText = literalExpression
+                ''"/var/lib/postgresql/''${config.package.psqlSchema}"'';
+              example = "/mnt/postgresql/11";
```

```suggestion
              example = "/mnt/postgresql/13";
```
we should match that with the manual